src/Terminus.php

```diff
@@ -0,0 +1,49 @@
+<?php
+// src/Terminus.php
+
+use Consolidation\AnnotatedCommand\AnnotatedCommand;
+use Consolidation\AnnotatedCommand\AnnotatedCommandFactory;
+use Consolidation\OutputFormatters\FormatterManager;
+use Symfony\Component\ClassLoader\Psr4ClassLoader;
+use Symfony\Component\Console\Application;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Yaml\Yaml;
+use Terminus\Config;
+
+class Terminus extends Application {
+
+  /**
+   * @inheritdoc
+   */
+  public function __construct() {
+    parent::__construct('Terminus', Config::get('version'));
+  }
+
+  /**
+   * @inheritdoc
+   */
+  protected function getDefaultHelperSet() {
```

Comment:
You're right, but I wanted to be clear for whomever is going to implement the helpers that this is how we'd do it.